doc/examples/nn_training.py

```diff
@@ -42,15 +43,15 @@
     optimized_phase = np.array(trained_model.layers[3].get_weights()[0])
 
     # Plot the result - optimized phase map
-    plotter = GeneratePropagationPlot(LightField(amp, optimized_phase), output_type=PlotTypes.PHASE)
+    plotter = Plotter(LightField(amp, optimized_phase), output_type=PlotTypes.PHASE)
```

Comment:
These lightfield objects needed to be plotted and used both here and in `GS.optimize` should be declared after amp/phase calculations

Reply:
so we can strongly emphasize that we are working on LF objects instead of some 2 dims arrays.